### PR TITLE
Updates to DG and Minor Bug fixes

### PR DESCRIPTION
**Pull request by jianhandev**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -218,7 +223,7 @@ This feature allows you to manage patient information. You can:
 3. Delete a patient 
 4. Edit a patient's details
 5. Add a remark to a patient
-6. Find a patient by his name
+6. Finds patients by name
```
Comment (thutahw) [inline docs/UserGuide.md]: Should be "Find patients by name" cuz it comes right after "You can:"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -512,10 +512,10 @@ Figure X. Workflow of an AddAppointment command with the help of CalendarManager
 
 #### 4.4.2. Current Implementation
 The `CalendarManager` class in the `Model` component contains an `AppointmentCalendar` object, storing the currently set 
-`year`, `month` and `day`. Note that the `year`, `month` and `day` attributes may not necessarily be storing the 
+year, month and day. Note that the year, month and day attributes may not necessarily be storing the
```
Comment (ReubenTeng) [inline docs/DeveloperGuide.md:515]: Should we leave the ` but change it to `Year`, `Month` and `Day` instead?

**Review by thutahw — APPROVED** [review]
LGTM, minor errors.

**Review by ReubenTeng — COMMENTED** [review]
Good simplification on UG/DG, but need to check on `year` `month` `day` formatting.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (kaitlynng) [reply docs/DeveloperGuide.md:515]: ^ agreed

**Review by kaitlynng — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ReubenTeng) [reply docs/DeveloperGuide.md:515]: @jianhandev was saying might look ugly (which is true) but maybe for consistent styling?

**Pull request merged**